### DISCUSSION TITLE
dwgtobim snapshot

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#6078] [Q&A]
now,in snapshot
when download the bridge AutoCAD DWG - 2.2.230.4 and dwg to  bim ,and the bim file just **have only one view,**
C:\Users\Administrator\AppData\Local\imodelbridges\RealDwgBridge\DwgBridgeM02.dll

but when delete the   bridge AutoCAD DWG - 2.2.230.4 , and dwg to bim, the bim file have more view and same views in AutoCAD software
 C:\Users\Administrator\AppData\Local\imodelbridges\IModelBridgeForMstn\DwgBridgeM02.dll

![2](https://github.com/iTwin/itwinjs-core/assets/18022720/2ab7642e-5d6a-4e36-8457-d1b7b2b6815f)
![1](https://github.com/iTwin/itwinjs-core/assets/18022720/161b9a98-f6f0-41fd-bca2-398c6473c770)

**Comment by donhfu** [comment]
Just ran into this post today....but currently a DWG connector creates views from model viewports, rather than views, from a DWG file.

I guess you would like to have named views in bim regardless if they are used or not?  If they are useful in iTwin workflow, they can be potentially supported.
